experiments/AMIP/modular/components/atmosphere/climaatmos_init.jl

```diff
@@ -109,16 +158,11 @@ function coupler_surface_setup(
 end
 
 """
-    compute_atmos_turbulent_fluxes!(atmos_sim::ClimaAtmosSimulation, csf)
+    get_new_p(atmos_sim::ClimaAtmosSimulation, csf)
```

Comment:
This name is not clear. Can we use `get_new_parameters`?

Reply:
It seems like this is returning the atmos cache rather than parameters, so `p` is more accurate. Maybe we could call it `get_new_cache` instead?

Reply:
sure!